packages/react-app-revamp/hooks/useProposal/index.ts

```diff
@@ -128,7 +128,7 @@ export function useProposal() {
 
       const proposalsIdsRawData = await getProposalIdsRaw(contractConfig, useLegacyGetAllProposalsIdFn, version);
 
-      let proposalsIds: Result;
+      let proposalsIds: any;
```

Comment:
was this type meant to get changed from Result to any?

Reply:
yeah, that type doesn't match old one anymore. i tested proposalIds but just with a single proposal, let me test with more so we can merge this one?

Reply:
oh gotcha, so is the import of Result on line 18 not used anymore/we could remove it? that's what made me confused about the use of any.

Reply:
ohhh yeah, we don't have to import it anymore. btw just checked contest with more entries on this deployment from mobile and all looks good!